app/gui/src/dashboard/pages/PaymentsSuccess.tsx

```diff
@@ -43,6 +43,7 @@ export function PaymentsSuccess() {
         } else {
           const timePassedMs = Number(new Date()) - startEpochMs
           if (timePassedMs > USER_REFETCH_TIMEOUT_MS) {
+            await router.push(DASHBOARD_PATH)
             throw new Error(
```

Comment:
Navigating does not reload the page, so once finished we still throw an error. I'm not sure what will be an effect of uncaught exception in `useMount` callback of probably destroyed component. If the intention is to display an error, maybe we should show it using `useToastAndLog`?